src/main/java/uk/gov/hmcts/juror/api/juror/service/JurorServiceImpl.java

```diff
@@ -149,12 +151,14 @@ && noThirdPartyPhoneNumbers(responseDto.getThirdParty())) {
             throw new NoPhoneNumberProvided();
         }
 
+        JurorPool jurorDetails = jurorPoolRepository.findByJurorJurorNumber(responseDto.getJurorNumber());
+        Juror juror = jurorDetails.getJuror();
```

Comment:
This wont work as the juror could be in multiple pools.

If this is only hit but the bureau officer you can use jurorPoolService.getJurorPoolFromUser(jurorNumber) if it is hit by the public too we will need a chat on this

Reply:
this does get hit by the public app 😄

Reply:
I just moved the call higher up in the method... we should raise another ticket to look into it if it causes problems?